### PR TITLE
test(wizard): tests for step error state

### PR DESCRIPTION
**Pull request by florin-borceanu**
Signed-off-by: Florin Borceanu <florin.borceanu@bytex.ro>

## PR Checklist

Please check if your PR fulfills the following requirements:

* [x] Tests for the changes have been added (for bug fixes / features)
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

* [ ] Bugfix
* [ ] Feature
* [ ] Code style update (formatting, local variables)
* [ ] Refactoring (no functional changes, no api changes)
* [ ] Build related changes
* [ ] CI related changes
* [ ] Documentation content changes
* [ ] clarity.design website / infrastructure changes
* [x] Other... Please describe:
Tests for wizard step error

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

* [ ] Yes
* [x] No

### CODE REVIEW

File: src/clr-angular/wizard/wizard-page.spec.ts [hunk 1]
```diff
@@ -483,6 +483,24 @@ export default function(): void {
...
+        });
+
+        it('should be true if hasError is true', () => {
+          testWizardPage.hasError = true;
+          fixture.detectChanges();
+          expect(testWizardPage.hasError).toBe(true);
+        });
+
+        it('should return false if hasError is false', () => {
+          testWizardPage.hasError = false;
```
Comment (Shijir) [inline src/clr-angular/wizard/wizard-page.spec.ts]: You are setting the value and testing it right afterward, why?

Comment (florin-borceanu) [reply src/clr-angular/wizard/wizard-page.spec.ts]: This PR is for #2203. The implementation was merged a few days ago and now I've done some tests for it. 

First, I made the tests after the `completed` template, and there was a test like this, but with two values. When I start writing, I made the test like there, but after that, I removed another variable and the tests remained as they are now. 
If you suggest, I'll remove or update those two tests.

Comment (Jinnie) [reply src/clr-angular/wizard/wizard-page.spec.ts]: Yeah, it's a bit strange now when the other variable is gone.
Please remove them if there are no side effects that need to be tested, events fired, etc.

**Comment by gnomeontherun** [comment]
@florin-borceanu can you rebase onto master to fixup tests I think?

**Comment by gnomeontherun** [comment]
@Shijir can you review or dismiss your block on the PR?

**Pull request merged**